fiat-web/src/main/java/com/netflix/spinnaker/fiat/controllers/AuthorizeController.java

```diff
@@ -188,7 +188,7 @@ public void getUserAuthorization(
     response.setStatus(HttpServletResponse.SC_NOT_FOUND);
   }
 
-  @RequestMapping(value = "/{userId:.+}/{resourceType:.+}", method = RequestMethod.POST)
+  @RequestMapping(value = "canCreate/{userId:.+}/{resourceType:.+}", method = RequestMethod.POST)
```

Comment:
This is a backwards incompatible change if something/someone is already calling this endpoint?

I don't believe `canCreate` belongs in the path personally ... 

What about overloading the existing `/authorize` with a `@POST` that takes the payload -- in this case I'd expect that we only support `authorization == create`?

```
@POST("/authorize/{userId}/{resourceType}/{resourceName}/{authorization}")
```

Thoughts on this @cfieber?

Reply:
👍

Reply:
Sounds good, I applied the suggestions